### DISCUSSION TITLE
Calling a url trough end used machine on Button Click and Displaying Response

### DISCUSSION THREAD

**Discussion by hismayilov** [Budibase/budibase#12988] [Help]
I want to call url, from users machine while the button is clicked.

**Comment by ConorWebb96** [comment]
I have no idea what you are asking here. Could you provide more information please. 

By URL do you mean a REST API request?

**Comment by hismayilov** [comment]
Hi @ConorWebb96 
scenario is like this, user authenticate www.url_auth.com with USB dongle, which injects certificate to windows, so when end user enter that URL, it asks ssl certificate selection, en user selects it, and response of this call is auth data, such as auth-token.
Afterwards end user will run REST API with that auth-token in header.
